lib/kubernetes-deploy/ejson_secret_provisioner.rb

```diff
@@ -189,14 +186,14 @@ def fetch_private_key_from_secret
 
     def run_kubectl_json(*args)
       args += ["--output=json"]
-      out, err, st = run_kubectl(*args)
+      out, err, st = kubectl.run(*args)
       raise EjsonSecretError, err unless st.success?
       result = JSON.parse(out)
       result.fetch('items', result)
     end
 
-    def run_kubectl(*args)
-      Kubectl.run_kubectl(*args, namespace: @namespace, context: @context, log_failure: false)
+    def kubectl
```

Comment:
I asked because there's `kubectl` method defined on the resource level and I didn't see why `EjsonSecretProvisioner` would redefine this method. `log_failure_by_default: false` is a good reason 👍